tools/stacktrace.toit

```diff
@@ -10,21 +10,26 @@ usage:
 
 OBJDUMP ::= "xtensa-esp32-elf-objdump"
 
+ELF_FILE ::= "path/to/toit.elf"
+
 main args/List:
-  parser := arguments.ArgumentParser
-  parser.describe_rest ["/path/to/toit.elf"]
-  parser.add_flag "disassemble" --short="d"
-  parser.add_option "objdump" --default=OBJDUMP
-  parser.add_option "backtrace" --default="-"
-  parsed := parser.parse args:
-    usage
-    exit 1
-  if args.size < 1: usage
+  parsed := null
+  parser := cli.Command "stacktrace"
+      --rest=[cli.OptionString --required ELF_FILE]
```

Comment:
Instead of calling it 'path/to/toit.elf', set the type to 'path', 'input-file' or 'in-file'
```suggestion
      --rest=[cli.OptionString --type='path' --required "elf-file"]
```

This would be printed as:
`<elf-file:path>`